package.json

```diff
@@ -269,7 +269,7 @@
     "eslint-plugin-react-native-a11y": "^3.3.0",
     "eslint-plugin-storybook": "^0.8.0",
     "eslint-plugin-testing-library": "^6.2.2",
-    "eslint-plugin-you-dont-need-lodash-underscore": "^6.12.0",
+    "eslint-plugin-you-dont-need-lodash-underscore": "6.12.0",
```

Comment:
Why change to be the specific version?

Reply:
After bumping `expensify-common`, a different version of this plugin was installed (6.14.0), which caused some unwanted eslint errors.